addons/google_calendar/models/calendar.py

```diff
@@ -47,7 +47,16 @@ def write(self, values):
         return super().write(values)
 
     def _get_sync_domain(self):
-        return [('partner_ids.user_ids', 'in', self.env.user.id)]
+        # in case of full sync, limit to 1y in past anyway
```

Comment:
As far I understand this, are we limiting sync to 1y in past and 1y in future ? In which case I would update comment accordingly.